src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -553,6 +553,10 @@ private void applyPatternImpl(String pattern) {
      * The string is constructed from internal information and therefore
      * does not necessarily equal the previously applied pattern.
      *
+     * @implSpec The implementation in {@link MessageFormat} returns a string
+     * that can be used to create a new instance that is semantically equivalent
+     * to this instance.
```

Comment:
The "can be used to create" seems conditional. Perhaps it can be worded as an assertion that can be tested.
```suggestion
     * @implSpec A new MessageFormat created from the string returned from the implementation of 
     * `MessageFormat.toPattern()` is semantically equivalent to this instance.
```

Reply:
> The "can be used to create" seems conditional. 

It is conditional - in the sense that you don't _have_ to use it to create a new instance of `MessageFormat`. You can also use it for something else, in other words.

But I also understand how it comes across as a bit wishy-washy...

Hmm, what do you think about this wording?

```
@implSpec The implementation in {@link MessageFormat} returns a string that,
when passed to the {@link MessageFormat(String)} constructor, produces an
instance that is semantically equivalent to this instance.
```

Reply:
Not sure which wording will ultimately be used, but if the wording ends up including the constructor, it's probably worth mentioning the `applyPattern` method as well.

Reply:
Good point... maybe this?
```
@implSpec The implementation in {@link MessageFormat} returns a string that, when passed
to a {@code MessageFormat()} constructor or {@link #applyPattern applyPattern()}, produces
an instance that is semantically equivalent to this instance.
```